src/components/Card/Card.js

```diff
@@ -1,30 +1,81 @@
 import { Tile } from "carbon-components-react";
 import React, { useState } from "react";
+import CardContent from "./CardContent/CardContent";
+import CardControls from "./CardControls/CardControls";
+import CardHeader from "./CardHeader/CardHeader";
+
+let currentValues = {
+  header: "Header",
+  content: "Content",
+};
```

Comment:
Try add card object data to application state and pass it to Card component, so once data updated in the card - they are updated in upstream state as well.